### ISSUE TITLE
GooglePay still instanciated

### ISSUE THREAD

**Issue by KevinOl** [datatrans/payment-button#7]
Hi again,

in my configuration I disabled googlePay. However, I keep on getting "ReferenceError: Can't find variable: PaymentRequest".
This originates from pay.js.

Below is my default init,

```
const defaultInit = {
	merchantName: 'test',
	useApplePay: true,
	useGooglePay: false,
	auto: false,
	allowedCardNetworks: [
		PAYMENT_METHOD_CREDITCARD_VISA,
		PAYMENT_METHOD_CREDITCARD_MASTERCARD,
		PAYMENT_METHOD_CREDITCARD_AMEX,
	],
	googlePayConfiguration: {
		buttonType: 'long',
		buttonStyle: 'black',
	},
	applePayConfiguration: {
		buttonType: 'plain',
		buttonStyle: 'black',
	},
}
```
Also, I am adding the merchantId later on,

```
window.PaymentButton.init({
				...defaultInit,
				merchantId,
                                 googlePayConfiguration: {
                                     ...defaultInit.googlePayConfiguration,
                                     merchantId, // why 2 different merchantIds?
                                 }
			})
```

This is my payment request,

```
details: {
		total: {
			label: 'Test label',
			amount: { value: amount, currency },
		},
	},
	options: {
		requestPayerEmail: false,
		requestPayerName: false,
		requestPayerPhone: false,
		requestShipping: false,
	},
	transaction: {
		countryCode: 'CH',
		refno,
	}
```

Seems I must have googlePay set as well...but I don't want. How to do that?

**Comment by dmengelt** [comment]
Hi @KevinOl 
Thanks for reporting this. We are looking into it and are coming back to you as soon as possible.
Regarding the second merchantId for Google Pay. This is just how it works with Google Pay. You need to pass the Google Pay related merchantId you obtained from the Google Pay merchant console there. This is not a Datatrans merchantId. Our docs say:

> Your google pay merchantId. Can be found in the Google Pay [merchant console](https://pay.google.com/business/console)

**Comment by fgrDtrx** [comment]
Hi @KevinOl 

The message you are getting `ReferenceError: Can't find variable: PaymentRequest` from pay.js comes from the google pay javascript. But since you disabled GooglePay it shouldn't be loaded in the first place. We will soon push a new version to also address this issue. I will let you know as soon as it is deployed.

**Comment by fgrDtrx** [comment]
Hi @KevinOl 

We have released a new version (still 1.0.0) which should fix your issue. Sorry it took us a while but the issue was far more reaching than we expected.
